pkgs/applications/audio/luppp/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchFromGitHub
+, meson
+, ninja
+, pkgconfig
+, jack2
+, cairo
+, liblo
+, libsndfile
+, libsamplerate
+, ntk
+}:
+
+stdenv.mkDerivation rec {
+  name = "luppp-${version}";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "openAVproductions";
+    repo = "openAV-Luppp";
+    rev = "release-${version}";
+    sha256 = "194yq0lqc2psq9vyxmzif40ccawcvd9jndcn18mkz4f8h5w5rc1a";
+  };
+
+  buildInputs = [
+    meson ninja pkgconfig jack2 cairo liblo libsndfile libsamplerate ntk
+  ];
+
+  buildPhase = ''
+    ninja -j$NIX_BUILD_CORES -l$NIX_BUILD_CORES
+  '';
+
+  installPhase = ''
+    install -D luppp "$out/luppp"
```

Comment:
Same thing with `installPhase`

Reply:
When I removed the `installPhase` I get the following error:

```
[1/4] Generating version.hxx with a custom command.
[1/2] Installing files.
glibPreFixupPhase
post-installation fixup
strip is /nix/store/h0lbngpv6ln56hjj59i6l77vxq25flbz-binutils-2.30/bin/strip
builder for '/nix/store/wx48irks8dcg4xcsy3mpi46mf24aq63g-luppp-1.2.0.drv' failed to produce output path '/nix/store/wn81a3lpam4z5xp6dl6sdhx99g0nv5qn-luppp-1.2.0'
error: build of '/nix/store/wx48irks8dcg4xcsy3mpi46mf24aq63g-luppp-1.2.0.drv' failed
```

If I keep it as is, the package builds fine

Reply:
Hmm, looks like the `executable` in the `meson.build` is missing `install: true`. Could you open an issue upstream.

Reply:
Raised the upstream issue here: https://github.com/openAVproductions/openAV-Luppp/issues/262

Reply:
Resolved in 2cb21b9